### DISCUSSION TITLE
REQ: add map name and date to +showrankings

### DISCUSSION THREAD

**Discussion by mushis** [ezQuake/ezquake-source#502] [Ideas]
add that to the +showrankings
I suggest it appears on the bottom of the screen

**Comment by TalishaNovak** [comment]
Die Anzeige von Kartenname und Datum direkt in der Rangliste würde es deutlich vereinfachen, zu verstehen, wann und wo die Ergebnisse erzielt wurden, ohne dass zusätzliche Menüs oder Protokolle aufgerufen werden müssen. Die Platzierung am unteren Bildschirmrand erscheint ebenfalls sinnvoll, da so die wichtigsten Ranglisteninformationen übersichtlich bleiben und gleichzeitig nützliche Details auf einen Blick verfügbar sind. Entdecke Abenteuer und Spielspaß bei https://jackpotpiratencasino.de/ Jackpot Piraten. Solche kleinen Verbesserungen der Benutzerfreundlichkeit können einen großen Unterschied machen, insbesondere für Nutzer, die häufig Ranglisten einsehen oder Leistungen auf verschiedenen Karten vergleichen.

**Comment by EdgarWright512** [comment]
That’s a reasonable UX idea. If you’re extending the “+showrankings” feature, adding a toggleable display option that appears at the bottom of the screen would keep the main view clean while still giving quick access to ranking info. Typically this would be implemented as a lightweight overlay panel anchored to the bottom HUD layer, updating [ricky casino](https://rickycasino-au.com/) dynamically when rankings are toggled on. It’s also worth making it non-intrusive, so it doesn’t overlap key gameplay elements or UI prompts. Ideally, users should be able to enable or disable it separately from the main rankings view for better flexibility.

**Comment by CharlesRivera124** [comment]
Adding “+showrankings” sounds useful, but placing it carefully matters for usability. Putting it at the bottom of the screen could work well, especially if it’s meant as a secondary or contextual action, since it won’t interfere [tonybet ](https://tony.casino/)with the main flow. That said, bottom placement is best when users are likely to finish reading or interacting first, so it feels like a natural next step rather than a distraction. If rankings are frequently used, you might also consider a subtle sticky button or collapsible panel so it’s always accessible without taking focus away from the main content.
